deposit/src/main/java/edu/unc/lib/deposit/work/DepositEmailHandler.java

```diff
@@ -54,8 +49,8 @@
 	@Autowired
 	private Dataset dataset;
 	
-	@Autowired
-	private FedoraAccessControlService accessControlService;
+	//@Autowired
```

Comment:
I should have put a comment in here, but this stuff is basically commented out as a reminder until we have access control worked out